pymodbus/__init__.py

```diff
@@ -3,17 +3,12 @@
 Released under the the BSD license
 """
 
-import logging as __logging
-from logging import NullHandler as __null
-
 import pymodbus.version as __version
+from ._logging import *
```

Comment:
hmmm a file named _logging is private and should surely not be imported in __init__.py

Just to be clear my obstruction if the file name.

I am pretty sure this will also cause a pylint problem, the asterisk.